performance/tests/test_eqprs_api.py

```diff
@@ -0,0 +1,56 @@
+import json
+from django.urls import reverse
+from rest_framework.test import APITestCase,APIRequestFactory
+from rest_framework import status
+from performance.views import IndicatorReadOnlyModelViewSet
+from performance.serializer import IndicatorSerializer
+from performance.models import Indicator
+import json
+import pprint
+
+class indicator_API_Test(APITestCase):
+    list_url = '/api/v1/'
+    fixture = ['test_api.json']
+    
+    def test_api(self):
+        response = self.client.get(self.list_url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+           
+        data = json.load(open("performance/test_api.json"))
+        response_data=json.dumps(data) 
+        dict_json=json.loads(response_data) 
```

Comment:
data and dict_json should be the same value - remember to delete your dev test code